src/app/loaders/express/error-handler.ts

```diff
@@ -115,14 +123,6 @@ const errorHandlerMiddlewares = (): (
     }
   }
 
-  // Assume 404 since no middleware responded
-  const catchNonExistentRoutesMiddleware: RequestHandler = function (
-    _req,
-    res,
-  ) {
-    res.sendStatus(StatusCodes.NOT_FOUND)
-  }
-
   return [genericErrorHandlerMiddleware, catchNonExistentRoutesMiddleware]
 }
 
```

Comment:
tiny nit: in an ideal world I would remove line 129 and change `errorHandlerMiddlewares` to a named export just so it's clear what the difference is between the multiple exports from this file. but not a big deal

Reply:
I'll do it!

Reply:
I wonder why the 2 handlers in `errorHandlerMiddlewares` were defined as local functions btw I'll convert them both to module functions for consistency.

Reply:
Done in https://github.com/opengovsg/FormSG/pull/5579/commits/5cc48bfe3cd4f037fab3ff12a7893e07656805d4